### PR TITLE
Lesson 4: FrontEnd

### PR DESCRIPTION
**Pull request by seemcat**
An HTML file with 3 inputs form: name, comment, and a submit button. When submit button is clicked, save comment into a file.

### CODE REVIEW

File: public/publicHTML.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Name: Maricris Comment: Hello world!
```
Comment (songz) [inline public/publicHTML.txt]: don't commit generated files.

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<form action = '/submit'>
```
Comment (songz) [inline public/index.html]: where is your whole html?

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+  Name: <input type = 'text' name = 'a1'/>
```
Comment (songz) [inline public/index.html]: `name='a1'` is a bad name Just like variables, please come up with better naming.

File: publicHTML.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const fs = require('fs');
+const app = express();
+
+app.listen(3002);
+app.use(express.static('public'));
+
+app.get('/submit', (req, res) => {
+  fs.appendFile('/home/mc/garage_script/public/publicHTML.txt', 'Name: ' + req.query.a1 + ' Comment: ' + req.query.a2 + '\n');
```
Comment (songz) [inline publicHTML.js]: `+` is really ugly for strings. use ``Name: ${req.query.a1}``

File: public/publicHTML.txt (hunk 1 above)
Comment (seemcat) [reply public/publicHTML.txt]: Gotcha!

File: public/index.html (hunk 2 above)
Comment (seemcat) [reply public/index.html]: *Added.

File: public/index.html (hunk 3 above)
Comment (seemcat) [reply public/index.html]: I agree, *changed.

File: publicHTML.js (hunk 4 above)
Comment (seemcat) [reply publicHTML.js]: Okay, *changed.

**Review by MadhuBeesetty — APPROVED** [review]

**Review by saumya665 — APPROVED** [review]

**Pull request merged**